apisix/balancer.lua

```diff
@@ -176,6 +177,17 @@ local function set_balancer_opts(route, ctx)
 end
 
 
+local function parse_server_for_upstream_host(picked_server, ctx)
+    local upstream_scheme = ctx.upstream_scheme
+    local standard_port = apisix_upstream.scheme_to_port[upstream_scheme]
```

Comment:
```suggestion
local function parse_server_for_upstream_host(picked_server, upstream_scheme)
    local standard_port = apisix_upstream.scheme_to_port[upstream_scheme]
```

is better?

Reply:
Yes.Thank you for your careful instruction